<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-11-29T14:00:30 IST" name="Regression Suite" finished-at="2022-11-29T14:00:39 IST" duration-ms="9120">
    <groups>
    </groups>
    <test started-at="2022-11-29T14:00:30 IST" name="Regression Test" finished-at="2022-11-29T14:00:39 IST" duration-ms="9120">
      <class name="ai.dalalstreet.testLayer.TestLoginPage">
        <test-method is-config="true" signature="start()[pri:0, instance:ai.dalalstreet.testLayer.TestLoginPage@515aebb0]" started-at="2022-11-29T14:00:30 IST" name="start" finished-at="2022-11-29T14:00:30 IST" duration-ms="179" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- start -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:ai.dalalstreet.testLayer.TestLoginPage@515aebb0]" started-at="2022-11-29T14:00:30 IST" name="setup" finished-at="2022-11-29T14:00:35 IST" duration-ms="5402" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyLoginpage()[pri:0, instance:ai.dalalstreet.testLayer.TestLoginPage@515aebb0]" started-at="2022-11-29T14:00:35 IST" name="verifyLoginpage" finished-at="2022-11-29T14:00:39 IST" duration-ms="3041" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://www.apps.dalalstreet.ai/dashboard] but found [https://apps.dalalstreet.ai/dashboard]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://www.apps.dalalstreet.ai/dashboard] but found [https://apps.dalalstreet.ai/dashboard]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at ai.dalalstreet.testLayer.TestLoginPage.verifyLoginpage(TestLoginPage.java:16)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginpage -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:ai.dalalstreet.testLayer.TestLoginPage@515aebb0]" started-at="2022-11-29T14:00:39 IST" name="tearDown" finished-at="2022-11-29T14:00:39 IST" duration-ms="235" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="end()[pri:0, instance:ai.dalalstreet.testLayer.TestLoginPage@515aebb0]" started-at="2022-11-29T14:00:39 IST" name="end" finished-at="2022-11-29T14:00:39 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- end -->
      </class> <!-- ai.dalalstreet.testLayer.TestLoginPage -->
    </test> <!-- Regression Test -->
  </suite> <!-- Regression Suite -->
</testng-results>
